06_homework_online_store/script.js

```diff
@@ -4,113 +4,106 @@ let bikes = [{
+                     "It first appeared at the 1974 Cologne motorcycle show, Intermot, and was dropped from the Honda range in 1978." +
+              "It had an air-cooled, transverse-mounted 408 cc inline four-cylinder engine"
     },
-
     {
         brand: "Yamaha",
         model: "SR400",
         engineCapacity: 400,
         colour: "black",
         imgSrc: 'images/yamaha.jpg',
-        price: 3500,
+        price:  "3500 Euro",
         condition: {
             year: 1997,
-            mileage: 23000
+            mileage: "23000 km"
         },
         inStock: true,
         description: "Single-cylinder, air-cooled, two-passenger motorcycles manufactured as a street version of the " +
-            "Yamaha XT500, with a standard riding posture and styling resembling the Universal Japanese Motorcycles " +
-            "of the 1970s. "
+                     "Yamaha XT500, with a standard riding posture and styling resembling the Universal Japanese Motorcycles " +
+                     "of the 1970s. "
     },
-
     {
         brand: "Kawasaki",
         model: "W650",
         engineCapacity: 650,
         colour: "black",
         imgSrc: 'images/kawasaki.jpg',
-        price: 3200,
+        price: "3200 Euro",
         condition: {
             year: 2003,
-            mileage: 13000
+            mileage: "13000 km"
         },
         inStock: true,
         description: "It is a retro standard motorcycle marketed by Kawasaki for model years 1999-2007" +
-            "In 1999,Kawasaki introduced the W650, resembling British motorcycles of the early 1960s, notably the Triumph Bonneville."
+                     "In 1999,Kawasaki introduced the W650, resembling British motorcycles of the early 1960s, notably the Triumph Bonneville."
     },
-
     {
         brand: "Triumph",
         model: "Thruxton",
         engineCapacity: 865,
         colour: "black",
         imgSrc: 'images/triumph.jpg',
-        price: 6000,
+        price: "6000 Euro",
         condition: {
             year: 2008,
-            mileage: 8000
+            mileage: "8000 km"
         },
         inStock: false,
         description: "The Triumph Thruxton is a series of British motorcycles with parallel-twin engines and sports styling." +
-            " The name Thruxton was first applied to a handbuilt machine for endurance racing in the mid 1960s, " +
-            "and later revived in the 2000s. Production 2004 - 2016"
+                     " The name Thruxton was first applied to a handbuilt machine for endurance racing in the mid 1960s, " +
+                     "and later revived in the 2000s. Production 2004 - 2016"
     }
 ];
-console.log(bikes);
-let i;
 
 function bikeDisplay() {
-    for (i = 0; i < bikes.length; i++) {
+    for (let i = 0; i < bikes.length; i++) {
         bikeCard = document.createElement("div");
         bikeCard.setAttribute("class", "card");
-        const main = document.getElementById('main')
```

Comment:
you deleted here main but below you're still using it :)

Reply:
I will be more attentive. Thank you! Fixed.